### DISCUSSION TITLE
How to copy multi  files?

### DISCUSSION THREAD

**Discussion by OriLiMu** [sxyazi/yazi#876] [1. Q&A] [answered]
I use space or v to select multi files then press `yy` and then `p`.

Just copy the first file I select why is this and how to solve it?

**Comment by sxyazi** [comment]
Please record a GIF so I can see your steps

**Reply by OriLiMu** [reply]
![Peek 2024-04-04 19-08](https://github.com/sxyazi/yazi/assets/11295355/3de7ae95-b93a-4ffb-9ef2-c2a85531ba69)

I press yy then p

**Reply by sxyazi** [reply]
Copy is `y`, not `yy`, https://yazi-rs.github.io/docs/quick-start#filedirectory-operations

**Reply by OriLiMu** [reply]
Tks. Another question plz. After I press p the copy done. Then the file 1 2 3 still selected.(highlight still on the left) 
But when I set the cursor on file 1_1 for example then press d. 1_1 get deleted not 1,2,3. So what is the meaning of the highlight here. Seems the file 1,2,3 not selected why the highlight still there.
![image](https://github.com/sxyazi/yazi/assets/11295355/b336d30d-0d91-4197-a038-27c369b56748)

**Reply by sxyazi** [reply, marked as answer]
When you press `y`, the files switch from a selected state to a yanked state. So, what you see isn't a selected state anymore, but rather the files being yanked (press `Y` to unyank). This means that at this point, you haven't selected any files, hence pressing `d` will do the action on the hovered single file.

**Reply by OriLiMu** [reply]
Anyway to remove the highlight mark after the highlight done? Or remove the  highlight by press `esc` manually. The highlight seems a little misleading for me

**Reply by sxyazi** [reply]
> press `Y` to unyank
